deepchem/models/losses.py

```diff
@@ -759,6 +762,133 @@ def log_sum_exp(x, axis=None):
     return y
 
 
+class GroverPretrainLoss(Loss):
+    """GroverPretrainLoss
```

Comment:
A quick reminder to add the loss to the docs

Reply:
added to docs in 0192870